src/schema/nmdc.yaml

```diff
@@ -1650,6 +1672,26 @@ classes:
           syntax: "{id_nmdc_prefix}:cspro-{id_shoulder}-{id_blade}{id_version}{id_locus}"
           interpolated: true
 
+  ReactionProcess:
```

Comment:
please give this a name that explicitly differentiates it from the exiting `ReactionClass` (which is about biological pathways)

I suggest `LaboratoryReactionPathway`

Reply:
https://microbiomedata.github.io/nmdc-schema/Reaction/ 

I think ReactionProcess is as far as we can get this right now. I could see arguments for changing Reaction too.
Let's not dwell on this term and track the issue as a todo

Reply:
Is class `ReactionProcess` not differentiated enough from class `Reaction`?

Reply:
Resolution: come up with a name descriptive to the fact that the process takes place in an experimental setting rather than being functional annotation